{"name": "test_01_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'Test_login_case' object has no attribute 'driver'", "trace": "self = <test_case.test_login.Test_login_case object at 0x000001EDCFF22198>\n\n    def test_01_login(self):\n        ''' 登录 '''\n>       lp = login_page(self.driver)\nE       AttributeError: 'Test_login_case' object has no attribute 'driver'\n\ntest_case\\test_login.py:17: AttributeError"}, "description": " 登录 ", "start": 1646363555082, "stop": 1646363555082, "uuid": "658093a5-40f8-4728-9a69-aca11856bc6a", "historyId": "4100a12eb90b1e54cf2161c6f3c20b6e", "testCaseId": "dfcaf0a329f939d784354277fc946a6a", "fullName": "test_case.test_login.Test_login_case#test_01_login", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login_case"}, {"name": "host", "value": "DESKTOP-GJB8D45"}, {"name": "thread", "value": "171752-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_login"}]}